WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -942,12 +982,27 @@ public void onPositiveClicked(@NonNull String instanceTag) {
             case TAG_QUICK_START_MIGRATION_DIALOG:
                 // TODO: Quick Start - Add analytics for migration dialog positive tapped.
                 break;
+            case TAG_REMOVE_NEXT_STEPS_DIALOG:
```

Comment:
```suggestion
            case TAG_REMOVE_NEXT_STEPS_DIALOG:
            // TODO: Quick Start - Add analytics for remove next steps dialog positive tapped.
```

We probably want analytics for tapping the positive button of the **_Remove Next Steps_** dialog just like the negative button [here](https://github.com/wordpress-mobile/WordPress-Android/pull/8813/files#diff-57a50fd5aaef95ac8ac5d2ebb2eb559eR1026).